bin/git-bulk

```diff
@@ -14,27 +14,66 @@ allwsmode=false
 #
 usage() {
   echo 1>&2 "usage: git bulk [-g] ([-a]|[-w <ws-name>]) <git command>"
+  echo 1>&2 "       git bulk --addproject <project-name>"
   echo 1>&2 "       git bulk --addworkspace <ws-name> <ws-root-directory>"
   echo 1>&2 "       git bulk --removeworkspace <ws-name>"
   echo 1>&2 "       git bulk --addcurrent <ws-name>"
   echo 1>&2 "       git bulk --purge"
   echo 1>&2 "       git bulk --listall"
 }
 
+function to_lowercase { echo $1 | tr [:upper:] [:lower:]; }
+
+# helper to get project name
+function getproject { git config --local --get project.name; }
```

Comment:
Is this function expected to be run under non-git repo? A workspace is not always a git repo. Maybe we need to add `2> /dev/null || echo ''` after it?

Reply:
well I have been always thinking in the context of git repositories, since it's all about git and git-bulk in a directory without `.git` in it is assumed out of scope by me. I'll take a more generic look at it later today!

Reply:
@spacewander I commited a small change to `--listall` which fixes this bug and the function returns all registered workspaces. So the API is not broken.
Now I should probably extend the function to return only workspaces of a given project / or current